<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Item</class>
 <widget class="QDialog" name="Item">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stock</string>
  </property>
  <widget class="QLabel" name="item_Label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Article :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="manufacturerCode_LineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="partNumber_LineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>341</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_LineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>271</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="name_Label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>59</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom :</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>501</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="uniqueIdentifier_CheckBox">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>80</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Identifiant</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_PushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="set_PushButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>260</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Définir</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="localisation_Label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Localication : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="localisation_LineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>271</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="maximum_Label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Maximum : </string>
   </property>
  </widget>
  <widget class="QSpinBox" name="maximum_SpinBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="minimum_Label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Minimum : </string>
   </property>
  </widget>
  <widget class="QSpinBox" name="minimum_SpinBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="quantity_PushButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Quantité</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>manufacturerCode_LineEdit</tabstop>
  <tabstop>partNumber_LineEdit</tabstop>
  <tabstop>name_LineEdit</tabstop>
  <tabstop>localisation_LineEdit</tabstop>
  <tabstop>maximum_SpinBox</tabstop>
  <tabstop>minimum_SpinBox</tabstop>
  <tabstop>cancel_PushButton</tabstop>
  <tabstop>set_PushButton</tabstop>
  <tabstop>uniqueIdentifier_CheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
